ironmon_tracker/data/DataHelper.lua

```diff
@@ -226,7 +250,7 @@ function DataHelper.buildTrackerScreenDisplay(forceView)
 
 		-- Update: Specific Moves
 		if move.id == 237 then -- 237 = Hidden Power
-			if data.x.viewingOwn then
+			if data.x.viewingOwn or useOpenBookInfo then
 				move.type = Tracker.getHiddenPowerType()
```

Comment:
Not sure why we are using `useOpenBookInfo` for hidden power type?

This functionality is for showing the user's manually-tracked hidden power type, or if there is none it returns the same `MoveData.HiddenPowerTypeList[1]` that the `else` block here does too.

If we're planning to let people manually set a type for enemy pokemon that doesn't seem like something to limit to the open book setting? Though realistically, in gen 3, we know trainers mostly have no IVs so like 90+% of trainers will have hidden power fighting anyways. If it were the case that they had varying IVs and the intent here would be to show the resulting hidden power type, that'd be info that you don't get from a log file (since it derives from IVs)

Reply:
Fixed this in https://github.com/besteon/Ironmon-Tracker/pull/404/commits/9bfba4ad6e24adcd1f9adc5ae1f6d08287c8d433